[2023-06-10T15:37:13.973+0000] {processor.py:157} INFO - Started process (PID=15358) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:37:13.975+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:37:13.976+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:37:13.976+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:37:13.993+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:37:13.989+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:37:13.993+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:37:14.013+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.044 seconds
[2023-06-10T15:37:44.492+0000] {processor.py:157} INFO - Started process (PID=15415) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:37:44.494+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:37:44.496+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:37:44.495+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:37:44.507+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:37:44.505+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:37:44.508+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:37:44.534+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.048 seconds
[2023-06-10T15:38:14.967+0000] {processor.py:157} INFO - Started process (PID=15472) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:38:14.968+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:38:14.969+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:38:14.969+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:38:14.974+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:38:14.973+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:38:14.974+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:38:14.988+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.024 seconds
[2023-06-10T15:38:45.461+0000] {processor.py:157} INFO - Started process (PID=15529) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:38:45.463+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:38:45.465+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:38:45.464+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:38:45.475+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:38:45.472+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:38:45.476+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:38:45.520+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.065 seconds
[2023-06-10T15:39:15.811+0000] {processor.py:157} INFO - Started process (PID=15586) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:39:15.813+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:39:15.813+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:39:15.813+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:39:15.818+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:39:15.817+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:39:15.819+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:39:15.836+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.027 seconds
[2023-06-10T15:39:46.503+0000] {processor.py:157} INFO - Started process (PID=15643) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:39:46.504+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:39:46.506+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:39:46.506+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:39:46.525+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:39:46.523+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:39:46.526+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:39:46.548+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.052 seconds
[2023-06-10T15:40:17.217+0000] {processor.py:157} INFO - Started process (PID=15700) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:40:17.219+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:40:17.221+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:40:17.221+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:40:17.228+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:40:17.226+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:40:17.230+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:40:17.253+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.040 seconds
[2023-06-10T15:40:48.085+0000] {processor.py:157} INFO - Started process (PID=15757) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:40:48.086+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:40:48.087+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:40:48.087+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:40:48.095+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:40:48.093+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:40:48.096+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:40:48.118+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.037 seconds
[2023-06-10T15:41:18.178+0000] {processor.py:157} INFO - Started process (PID=15815) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:41:18.179+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:41:18.181+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:41:18.181+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:41:18.189+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:41:18.187+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:41:18.189+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:41:18.208+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.037 seconds
[2023-06-10T15:41:48.861+0000] {processor.py:157} INFO - Started process (PID=15872) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:41:48.863+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:41:48.864+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:41:48.864+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:41:48.871+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:41:48.870+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:41:48.872+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:41:48.891+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.040 seconds
[2023-06-10T15:42:19.213+0000] {processor.py:157} INFO - Started process (PID=15929) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:42:19.216+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:42:19.217+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:42:19.217+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:42:19.226+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:42:19.224+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:42:19.227+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:42:19.250+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.042 seconds
[2023-06-10T15:42:49.721+0000] {processor.py:157} INFO - Started process (PID=15986) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:42:49.722+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:42:49.723+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:42:49.723+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:42:49.734+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:42:49.729+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:42:49.735+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:42:49.753+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.037 seconds
[2023-06-10T15:43:19.994+0000] {processor.py:157} INFO - Started process (PID=16043) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:43:19.997+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:43:19.998+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:43:19.998+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:43:20.005+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:43:20.003+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:43:20.006+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:43:20.031+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.041 seconds
[2023-06-10T15:43:50.121+0000] {processor.py:157} INFO - Started process (PID=16100) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:43:50.122+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:43:50.123+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:43:50.123+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:43:50.131+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:43:50.130+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:43:50.132+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:43:50.150+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.033 seconds
[2023-06-10T15:44:20.215+0000] {processor.py:157} INFO - Started process (PID=16157) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:44:20.216+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:44:20.216+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:44:20.216+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:44:20.222+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:44:20.221+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:44:20.222+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:44:20.237+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.025 seconds
[2023-06-10T15:44:50.973+0000] {processor.py:157} INFO - Started process (PID=16214) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:44:50.974+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:44:50.975+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:44:50.975+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:44:50.980+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:44:50.979+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:44:50.981+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:44:50.995+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.025 seconds
[2023-06-10T15:45:21.614+0000] {processor.py:157} INFO - Started process (PID=16271) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:45:21.615+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:45:21.616+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:45:21.615+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:45:21.623+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:45:21.619+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:45:21.623+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:45:21.639+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.028 seconds
[2023-06-10T15:45:52.186+0000] {processor.py:157} INFO - Started process (PID=16328) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:45:52.187+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:45:52.188+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:45:52.188+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:45:52.197+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:45:52.195+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:45:52.199+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:45:52.222+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.041 seconds
[2023-06-10T15:46:22.917+0000] {processor.py:157} INFO - Started process (PID=16385) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:46:22.919+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:46:22.921+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:46:22.921+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:46:22.933+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:46:22.931+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:46:22.933+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:46:22.959+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.050 seconds
[2023-06-10T15:46:53.334+0000] {processor.py:157} INFO - Started process (PID=16442) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:46:53.335+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:46:53.336+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:46:53.336+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:46:53.341+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:46:53.340+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:46:53.341+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:46:53.353+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.022 seconds
[2023-06-10T15:47:23.898+0000] {processor.py:157} INFO - Started process (PID=16499) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:47:23.899+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:47:23.900+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:47:23.900+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:47:23.905+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:47:23.903+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:47:23.906+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:47:23.923+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.027 seconds
[2023-06-10T15:47:54.918+0000] {processor.py:157} INFO - Started process (PID=16556) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:47:54.919+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:47:54.919+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:47:54.919+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:47:54.924+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:47:54.923+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:47:54.925+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:47:54.938+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.023 seconds
[2023-06-10T15:48:25.488+0000] {processor.py:157} INFO - Started process (PID=16613) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:48:25.489+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:48:25.490+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:48:25.490+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:48:25.495+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:48:25.494+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:48:25.497+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:48:25.513+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.027 seconds
[2023-06-10T15:48:55.654+0000] {processor.py:157} INFO - Started process (PID=16670) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:48:55.656+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:48:55.656+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:48:55.656+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:48:55.662+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:48:55.660+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:48:55.662+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:48:55.676+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.025 seconds
[2023-06-10T15:49:26.202+0000] {processor.py:157} INFO - Started process (PID=16727) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:49:26.203+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:49:26.204+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:49:26.204+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:49:26.213+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:49:26.210+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:49:26.213+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:49:26.228+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.028 seconds
[2023-06-10T15:49:56.907+0000] {processor.py:157} INFO - Started process (PID=16784) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:49:56.908+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:49:56.908+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:49:56.908+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:49:56.914+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:49:56.912+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:49:56.915+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:49:56.930+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.026 seconds
[2023-06-10T15:50:28.059+0000] {processor.py:157} INFO - Started process (PID=16841) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:50:28.060+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:50:28.061+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:50:28.060+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:50:28.066+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:50:28.064+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:50:28.066+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:50:28.080+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.024 seconds
[2023-06-10T15:50:58.788+0000] {processor.py:157} INFO - Started process (PID=16898) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:50:58.789+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:50:58.789+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:50:58.789+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:50:58.794+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:50:58.793+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:50:58.795+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:50:58.809+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.024 seconds
[2023-06-10T15:51:30.693+0000] {processor.py:157} INFO - Started process (PID=16955) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:51:30.695+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:51:30.696+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:51:30.696+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:51:30.706+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:51:30.703+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:51:30.707+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:51:30.727+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.039 seconds
[2023-06-10T15:52:01.217+0000] {processor.py:157} INFO - Started process (PID=17012) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:52:01.218+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:52:01.219+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:52:01.219+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:52:01.224+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:52:01.223+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:52:01.224+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:52:01.236+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.022 seconds
[2023-06-10T15:52:32.473+0000] {processor.py:157} INFO - Started process (PID=17069) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:52:32.474+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:52:32.474+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:52:32.474+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:52:32.479+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:52:32.478+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:52:32.480+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:52:32.494+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.024 seconds
[2023-06-10T15:53:03.488+0000] {processor.py:157} INFO - Started process (PID=17126) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:53:03.489+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:53:03.489+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:53:03.489+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:53:03.494+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:53:03.493+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:53:03.494+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:53:03.509+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.024 seconds
[2023-06-10T15:53:35.292+0000] {processor.py:157} INFO - Started process (PID=17183) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:53:35.293+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:53:35.294+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:53:35.294+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:53:35.300+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:53:35.298+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:53:35.300+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:53:35.313+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.024 seconds
[2023-06-10T15:54:06.827+0000] {processor.py:157} INFO - Started process (PID=17241) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:54:06.829+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:54:06.829+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:54:06.829+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:54:06.839+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:54:06.837+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:54:06.840+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:54:06.875+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.051 seconds
[2023-06-10T15:54:37.708+0000] {processor.py:157} INFO - Started process (PID=17300) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:54:37.709+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:54:37.710+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:54:37.710+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:54:37.729+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:54:37.723+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:54:37.731+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:54:37.751+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.046 seconds
[2023-06-10T15:55:08.343+0000] {processor.py:157} INFO - Started process (PID=17358) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:55:08.344+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:55:08.345+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:55:08.345+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:55:08.353+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:55:08.352+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:55:08.353+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:55:08.370+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.031 seconds
[2023-06-10T15:55:39.527+0000] {processor.py:157} INFO - Started process (PID=17415) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:55:39.527+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:55:39.528+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:55:39.528+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:55:39.533+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:55:39.532+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:55:39.533+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:55:39.549+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.025 seconds
[2023-06-10T15:56:10.387+0000] {processor.py:157} INFO - Started process (PID=17471) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:56:10.388+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:56:10.388+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:56:10.388+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:56:10.393+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:56:10.392+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:56:10.393+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:56:10.406+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.022 seconds
[2023-06-10T15:56:41.789+0000] {processor.py:157} INFO - Started process (PID=17528) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:56:41.790+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:56:41.791+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:56:41.791+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:56:41.798+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:56:41.797+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:56:41.799+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:56:41.811+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.027 seconds
[2023-06-10T15:57:13.584+0000] {processor.py:157} INFO - Started process (PID=17585) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:57:13.585+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:57:13.586+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:57:13.586+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:57:13.592+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:57:13.590+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:57:13.592+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:57:13.612+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.032 seconds
[2023-06-10T15:57:44.979+0000] {processor.py:157} INFO - Started process (PID=17649) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:57:44.980+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:57:44.981+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:57:44.981+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:57:44.986+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:57:44.985+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:57:44.986+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:57:45.000+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.023 seconds
[2023-06-10T15:58:16.107+0000] {processor.py:157} INFO - Started process (PID=17707) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:58:16.108+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:58:16.109+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:58:16.109+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:58:16.114+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:58:16.112+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:58:16.114+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:58:16.131+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.027 seconds
[2023-06-10T15:58:47.621+0000] {processor.py:157} INFO - Started process (PID=17767) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:58:47.622+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:58:47.622+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:58:47.622+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:58:47.627+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:58:47.626+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:58:47.628+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:58:47.644+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.026 seconds
[2023-06-10T15:59:18.289+0000] {processor.py:157} INFO - Started process (PID=17825) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:59:18.291+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:59:18.292+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:59:18.292+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:59:18.299+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:59:18.298+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:59:18.300+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:59:18.317+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.033 seconds
[2023-06-10T15:59:50.138+0000] {processor.py:157} INFO - Started process (PID=17888) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T15:59:50.141+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T15:59:50.142+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:59:50.142+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:59:50.156+0000] {logging_mixin.py:149} INFO - [2023-06-10T15:59:50.154+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T15:59:50.158+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T15:59:50.182+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.050 seconds
[2023-06-10T16:00:22.109+0000] {processor.py:157} INFO - Started process (PID=17957) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:00:22.111+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:00:22.112+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:00:22.112+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:00:22.118+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:00:22.116+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:00:22.118+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:00:22.132+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.026 seconds
[2023-06-10T16:00:52.757+0000] {processor.py:157} INFO - Started process (PID=18017) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:00:52.760+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:00:52.760+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:00:52.760+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:00:52.767+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:00:52.766+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:00:52.767+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:00:52.781+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.026 seconds
[2023-06-10T16:01:23.371+0000] {processor.py:157} INFO - Started process (PID=18074) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:01:23.373+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:01:23.374+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:01:23.373+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:01:23.379+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:01:23.378+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:01:23.379+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:01:23.393+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.024 seconds
[2023-06-10T16:01:55.514+0000] {processor.py:157} INFO - Started process (PID=18130) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:01:55.517+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:01:55.517+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:01:55.517+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:01:55.523+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:01:55.521+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:01:55.523+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:01:55.538+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.027 seconds
[2023-06-10T16:02:27.434+0000] {processor.py:157} INFO - Started process (PID=18187) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:02:27.436+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:02:27.437+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:02:27.437+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:02:27.443+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:02:27.442+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:02:27.444+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:02:27.457+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.027 seconds
[2023-06-10T16:02:57.862+0000] {processor.py:157} INFO - Started process (PID=18244) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:02:57.863+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:02:57.864+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:02:57.864+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:02:57.869+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:02:57.868+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:02:57.869+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:02:57.884+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.025 seconds
[2023-06-10T16:03:29.185+0000] {processor.py:157} INFO - Started process (PID=18301) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:03:29.188+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:03:29.189+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:03:29.189+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:03:29.196+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:03:29.194+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:03:29.196+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:03:29.211+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.030 seconds
[2023-06-10T16:04:01.748+0000] {processor.py:157} INFO - Started process (PID=18358) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:04:01.749+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:04:01.751+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:04:01.751+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:04:01.761+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:04:01.759+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:04:01.761+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:04:01.780+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.037 seconds
[2023-06-10T16:04:31.990+0000] {processor.py:157} INFO - Started process (PID=18414) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:04:31.991+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:04:31.992+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:04:31.992+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:04:31.997+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:04:31.996+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:04:31.997+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:04:32.012+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.025 seconds
[2023-06-10T16:05:04.057+0000] {processor.py:157} INFO - Started process (PID=18472) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:05:04.058+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:05:04.059+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:05:04.059+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:05:04.064+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:05:04.063+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:05:04.065+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:05:04.079+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.025 seconds
[2023-06-10T16:05:35.293+0000] {processor.py:157} INFO - Started process (PID=18529) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:05:35.294+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:05:35.295+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:05:35.295+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:05:35.302+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:05:35.300+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:05:35.303+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:05:35.336+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.046 seconds
[2023-06-10T16:06:06.953+0000] {processor.py:157} INFO - Started process (PID=18586) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:06:06.954+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:06:06.954+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:06:06.954+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:06:06.959+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:06:06.957+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:06:06.959+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:06:06.971+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.021 seconds
[2023-06-10T16:06:38.108+0000] {processor.py:157} INFO - Started process (PID=18643) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:06:38.110+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:06:38.110+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:06:38.110+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:06:38.116+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:06:38.115+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:06:38.117+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:06:38.134+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.029 seconds
[2023-06-10T16:07:08.533+0000] {processor.py:157} INFO - Started process (PID=18700) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:07:08.534+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:07:08.534+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:07:08.534+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:07:08.539+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:07:08.538+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:07:08.539+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:07:08.552+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.022 seconds
[2023-06-10T16:07:39.823+0000] {processor.py:157} INFO - Started process (PID=18757) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:07:39.824+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:07:39.825+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:07:39.825+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:07:39.829+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:07:39.828+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:07:39.830+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:07:39.845+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.025 seconds
[2023-06-10T16:08:10.276+0000] {processor.py:157} INFO - Started process (PID=18814) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:08:10.277+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:08:10.277+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:08:10.277+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:08:10.282+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:08:10.281+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:08:10.283+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:08:10.298+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.024 seconds
[2023-06-10T16:08:41.611+0000] {processor.py:157} INFO - Started process (PID=18871) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:08:41.612+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:08:41.613+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:08:41.613+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:08:41.618+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:08:41.617+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:08:41.618+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:08:41.640+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.031 seconds
[2023-06-10T16:09:13.573+0000] {processor.py:157} INFO - Started process (PID=18928) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:09:13.574+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:09:13.575+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:09:13.575+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:09:13.580+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:09:13.579+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:09:13.581+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:09:13.596+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.027 seconds
[2023-06-10T16:09:44.381+0000] {processor.py:157} INFO - Started process (PID=18985) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:09:44.382+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:09:44.383+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:09:44.383+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:09:44.389+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:09:44.388+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:09:44.390+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:09:44.408+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.030 seconds
[2023-06-10T16:10:15.199+0000] {processor.py:157} INFO - Started process (PID=19042) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:10:15.200+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:10:15.201+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:10:15.201+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:10:15.206+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:10:15.204+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:10:15.206+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:10:15.221+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.030 seconds
[2023-06-10T16:10:46.494+0000] {processor.py:157} INFO - Started process (PID=19099) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:10:46.496+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:10:46.496+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:10:46.496+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:10:46.500+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:10:46.499+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:10:46.501+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:10:46.513+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.022 seconds
[2023-06-10T16:11:18.499+0000] {processor.py:157} INFO - Started process (PID=19163) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:11:18.500+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:11:18.501+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:11:18.501+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:11:18.506+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:11:18.505+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:11:18.507+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:11:18.521+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.026 seconds
[2023-06-10T16:11:49.400+0000] {processor.py:157} INFO - Started process (PID=19220) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:11:49.401+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:11:49.402+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:11:49.402+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:11:49.406+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:11:49.405+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:11:49.406+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:11:49.421+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.023 seconds
[2023-06-10T16:12:20.909+0000] {processor.py:157} INFO - Started process (PID=19277) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:12:20.911+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:12:20.912+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:12:20.912+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:12:20.921+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:12:20.917+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:12:20.922+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:12:20.938+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.033 seconds
[2023-06-10T16:12:51.940+0000] {processor.py:157} INFO - Started process (PID=19335) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:12:51.941+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:12:51.941+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:12:51.941+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:12:51.946+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:12:51.945+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:12:51.946+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:12:51.961+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.024 seconds
[2023-06-10T16:13:23.323+0000] {processor.py:157} INFO - Started process (PID=19392) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:13:23.324+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:13:23.325+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:13:23.325+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:13:23.331+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:13:23.329+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:13:23.331+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:13:23.351+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.032 seconds
[2023-06-10T16:13:55.022+0000] {processor.py:157} INFO - Started process (PID=19452) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:13:55.022+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:13:55.023+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:13:55.023+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:13:55.027+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:13:55.026+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:13:55.028+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:13:55.041+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.022 seconds
[2023-06-10T16:14:26.314+0000] {processor.py:157} INFO - Started process (PID=19511) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:14:26.316+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:14:26.316+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:14:26.316+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:14:26.321+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:14:26.320+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:14:26.321+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:14:26.341+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.029 seconds
[2023-06-10T16:14:57.619+0000] {processor.py:157} INFO - Started process (PID=19568) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:14:57.620+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:14:57.629+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:14:57.629+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:14:57.640+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:14:57.638+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:14:57.641+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:14:57.656+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.040 seconds
[2023-06-10T16:15:28.934+0000] {processor.py:157} INFO - Started process (PID=19625) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:15:28.935+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:15:28.936+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:15:28.936+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:15:28.941+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:15:28.940+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:15:28.941+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:15:28.958+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.026 seconds
[2023-06-10T16:16:00.163+0000] {processor.py:157} INFO - Started process (PID=19682) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:16:00.164+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:16:00.165+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:16:00.165+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:16:00.170+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:16:00.169+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:16:00.170+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:16:00.185+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.025 seconds
[2023-06-10T16:16:31.213+0000] {processor.py:157} INFO - Started process (PID=19739) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:16:31.214+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:16:31.215+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:16:31.215+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:16:31.219+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:16:31.218+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:16:31.220+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:16:31.234+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.024 seconds
[2023-06-10T16:17:02.886+0000] {processor.py:157} INFO - Started process (PID=19796) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:17:02.887+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:17:02.887+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:17:02.887+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:17:02.892+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:17:02.891+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:17:02.893+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:17:02.909+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.025 seconds
[2023-06-10T16:17:34.500+0000] {processor.py:157} INFO - Started process (PID=19853) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:17:34.502+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:17:34.503+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:17:34.502+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:17:34.508+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:17:34.506+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:17:34.509+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:17:34.533+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.038 seconds
[2023-06-10T16:18:05.078+0000] {processor.py:157} INFO - Started process (PID=19910) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:18:05.080+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:18:05.081+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:18:05.081+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:18:05.088+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:18:05.086+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:18:05.090+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:18:05.114+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.040 seconds
[2023-06-10T16:18:36.399+0000] {processor.py:157} INFO - Started process (PID=19967) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:18:36.400+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:18:36.400+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:18:36.400+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:18:36.409+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:18:36.407+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:18:36.409+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:18:36.427+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.032 seconds
[2023-06-10T16:19:07.029+0000] {processor.py:157} INFO - Started process (PID=20024) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:19:07.030+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:19:07.031+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:19:07.031+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:19:07.036+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:19:07.034+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:19:07.036+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:19:07.049+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.024 seconds
[2023-06-10T16:19:39.307+0000] {processor.py:157} INFO - Started process (PID=20081) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:19:39.308+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:19:39.309+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:19:39.309+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:19:39.313+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:19:39.312+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:19:39.313+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:19:39.326+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.022 seconds
[2023-06-10T16:20:09.921+0000] {processor.py:157} INFO - Started process (PID=20138) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:20:09.922+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:20:09.922+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:20:09.922+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:20:09.927+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:20:09.926+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:20:09.927+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:20:09.940+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.022 seconds
[2023-06-10T16:20:42.499+0000] {processor.py:157} INFO - Started process (PID=20197) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:20:42.501+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:20:42.501+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:20:42.501+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:20:42.508+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:20:42.506+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:20:42.508+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:20:42.538+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.041 seconds
[2023-06-10T16:21:12.944+0000] {processor.py:157} INFO - Started process (PID=20254) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:21:12.949+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:21:12.950+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:21:12.950+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:21:12.956+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:21:12.955+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
ModuleNotFoundError: No module named 'modules'
[2023-06-10T16:21:12.957+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:21:12.973+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.033 seconds
[2023-06-10T16:21:43.774+0000] {processor.py:157} INFO - Started process (PID=20312) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:21:43.775+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:21:43.776+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:21:43.776+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:21:44.247+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:21:44.239+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
  File "/home/airflow/airflow_hw/modules/pipeline.py", line 7, in <module>
    from sklearn.compose import ColumnTransformer, make_column_selector
ModuleNotFoundError: No module named 'sklearn'
[2023-06-10T16:21:44.249+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:21:44.293+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.521 seconds
[2023-06-10T16:22:15.310+0000] {processor.py:157} INFO - Started process (PID=20380) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:22:15.311+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:22:15.312+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:22:15.312+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:22:15.598+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:22:15.594+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
  File "/home/airflow/airflow_hw/modules/pipeline.py", line 7, in <module>
    from sklearn.compose import ColumnTransformer, make_column_selector
ModuleNotFoundError: No module named 'sklearn'
[2023-06-10T16:22:15.599+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:22:15.613+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.306 seconds
[2023-06-10T16:22:46.682+0000] {processor.py:157} INFO - Started process (PID=20442) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:22:46.683+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:22:46.684+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:22:46.684+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:22:47.035+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:22:47.031+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
  File "/home/airflow/airflow_hw/modules/pipeline.py", line 7, in <module>
    from sklearn.compose import ColumnTransformer, make_column_selector
ModuleNotFoundError: No module named 'sklearn'
[2023-06-10T16:22:47.036+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:22:47.053+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.373 seconds
[2023-06-10T16:23:17.090+0000] {processor.py:157} INFO - Started process (PID=20505) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:23:17.092+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:23:17.092+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:23:17.092+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:23:17.350+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:23:17.347+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
  File "/home/airflow/airflow_hw/modules/pipeline.py", line 7, in <module>
    from sklearn.compose import ColumnTransformer, make_column_selector
ModuleNotFoundError: No module named 'sklearn'
[2023-06-10T16:23:17.350+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:23:17.364+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.276 seconds
[2023-06-10T16:23:48.027+0000] {processor.py:157} INFO - Started process (PID=20567) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:23:48.029+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:23:48.030+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:23:48.030+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:23:48.269+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:23:48.264+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
  File "/home/airflow/airflow_hw/modules/pipeline.py", line 7, in <module>
    from sklearn.compose import ColumnTransformer, make_column_selector
ModuleNotFoundError: No module named 'sklearn'
[2023-06-10T16:23:48.270+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:23:48.287+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.264 seconds
[2023-06-10T16:24:18.724+0000] {processor.py:157} INFO - Started process (PID=20628) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:24:18.725+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:24:18.726+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:24:18.726+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:24:18.981+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:24:18.977+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
  File "/home/airflow/airflow_hw/modules/pipeline.py", line 7, in <module>
    from sklearn.compose import ColumnTransformer, make_column_selector
ModuleNotFoundError: No module named 'sklearn'
[2023-06-10T16:24:19.008+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:24:19.022+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.300 seconds
[2023-06-10T16:24:49.354+0000] {processor.py:157} INFO - Started process (PID=20689) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:24:49.358+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:24:49.360+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:24:49.360+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:24:49.757+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:24:49.747+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
  File "/home/airflow/airflow_hw/modules/pipeline.py", line 7, in <module>
    from sklearn.compose import ColumnTransformer, make_column_selector
ModuleNotFoundError: No module named 'sklearn'
[2023-06-10T16:24:49.758+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:24:49.777+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.427 seconds
[2023-06-10T16:25:19.869+0000] {processor.py:157} INFO - Started process (PID=20750) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:25:19.870+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:25:19.871+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:25:19.871+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:25:20.149+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:25:20.144+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
  File "/home/airflow/airflow_hw/modules/pipeline.py", line 7, in <module>
    from sklearn.compose import ColumnTransformer, make_column_selector
ModuleNotFoundError: No module named 'sklearn'
[2023-06-10T16:25:20.149+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:25:20.165+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.299 seconds
[2023-06-10T16:25:50.599+0000] {processor.py:157} INFO - Started process (PID=20811) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:25:50.600+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:25:50.601+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:25:50.601+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:25:50.918+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:25:50.915+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
  File "/home/airflow/airflow_hw/modules/pipeline.py", line 7, in <module>
    from sklearn.compose import ColumnTransformer, make_column_selector
ModuleNotFoundError: No module named 'sklearn'
[2023-06-10T16:25:50.919+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:25:50.935+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.338 seconds
[2023-06-10T16:26:21.293+0000] {processor.py:157} INFO - Started process (PID=20872) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:26:21.294+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:26:21.294+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:26:21.294+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:26:21.549+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:26:21.546+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
  File "/home/airflow/airflow_hw/modules/pipeline.py", line 7, in <module>
    from sklearn.compose import ColumnTransformer, make_column_selector
ModuleNotFoundError: No module named 'sklearn'
[2023-06-10T16:26:21.550+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:26:21.563+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.273 seconds
[2023-06-10T16:26:52.436+0000] {processor.py:157} INFO - Started process (PID=20933) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:26:52.437+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:26:52.438+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:26:52.438+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:26:52.722+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:26:52.717+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
  File "/home/airflow/airflow_hw/modules/pipeline.py", line 7, in <module>
    from sklearn.compose import ColumnTransformer, make_column_selector
ModuleNotFoundError: No module named 'sklearn'
[2023-06-10T16:26:52.724+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:26:52.740+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.306 seconds
[2023-06-10T16:27:23.106+0000] {processor.py:157} INFO - Started process (PID=21003) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:27:23.107+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:27:23.108+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:27:23.108+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:27:23.405+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:27:23.401+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
  File "/home/airflow/airflow_hw/modules/pipeline.py", line 7, in <module>
    from sklearn.compose import ColumnTransformer, make_column_selector
ModuleNotFoundError: No module named 'sklearn'
[2023-06-10T16:27:23.406+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:27:23.423+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.321 seconds
[2023-06-10T16:27:53.660+0000] {processor.py:157} INFO - Started process (PID=21067) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:27:53.661+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:27:53.662+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:27:53.662+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:27:54.122+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:27:54.100+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
  File "/home/airflow/airflow_hw/modules/pipeline.py", line 7, in <module>
    from sklearn.compose import ColumnTransformer, make_column_selector
ModuleNotFoundError: No module named 'sklearn'
[2023-06-10T16:27:54.126+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:27:54.194+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.538 seconds
[2023-06-10T16:28:24.287+0000] {processor.py:157} INFO - Started process (PID=21134) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:28:24.288+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:28:24.289+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:28:24.289+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:28:24.541+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:28:24.536+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/hw_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/hw_dag.py", line 14, in <module>
    from modules.pipeline import pipeline
  File "/home/airflow/airflow_hw/modules/pipeline.py", line 7, in <module>
    from sklearn.compose import ColumnTransformer, make_column_selector
ModuleNotFoundError: No module named 'sklearn'
[2023-06-10T16:28:24.541+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:28:24.560+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.276 seconds
[2023-06-10T16:28:54.883+0000] {processor.py:157} INFO - Started process (PID=21196) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:28:54.884+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:28:54.885+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:28:54.885+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:28:55.757+0000] {processor.py:836} INFO - DAG(s) dict_keys(['car_price_prediction']) retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:28:55.899+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:28:55.899+0000] {manager.py:504} INFO - Created Permission View: can delete on DAG:car_price_prediction
[2023-06-10T16:28:55.904+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:28:55.904+0000] {manager.py:504} INFO - Created Permission View: can read on DAG:car_price_prediction
[2023-06-10T16:28:55.907+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:28:55.906+0000] {manager.py:504} INFO - Created Permission View: can edit on DAG:car_price_prediction
[2023-06-10T16:28:55.907+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:28:55.907+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-06-10T16:28:55.913+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:28:55.913+0000] {dag.py:2747} INFO - Creating ORM DAG for car_price_prediction
[2023-06-10T16:28:55.918+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:28:55.918+0000] {dag.py:3490} INFO - Setting next_dagrun for car_price_prediction to 2022-06-10T15:00:00+00:00, run_after=2022-06-11T15:00:00+00:00
[2023-06-10T16:28:55.926+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 1.045 seconds
[2023-06-10T16:29:26.954+0000] {processor.py:157} INFO - Started process (PID=21264) to work on /opt/airflow/dags/hw_dag.py
[2023-06-10T16:29:26.956+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/hw_dag.py for tasks to queue
[2023-06-10T16:29:26.958+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:29:26.958+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:29:27.758+0000] {processor.py:836} INFO - DAG(s) dict_keys(['car_price_prediction']) retrieved from /opt/airflow/dags/hw_dag.py
[2023-06-10T16:29:27.807+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:29:27.806+0000] {dag.py:2726} INFO - Sync 1 DAGs
[2023-06-10T16:29:27.823+0000] {logging_mixin.py:149} INFO - [2023-06-10T16:29:27.823+0000] {dag.py:3490} INFO - Setting next_dagrun for car_price_prediction to 2022-06-10T15:00:00+00:00, run_after=2022-06-11T15:00:00+00:00
[2023-06-10T16:29:27.833+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/hw_dag.py took 0.885 seconds
